sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClient.cs

```diff
@@ -16,6 +16,7 @@ namespace Azure.Data.AppConfiguration
     /// </summary>
     public partial class ConfigurationClient
     {
+        private const string OTelAttributeKey = "az.app_configuration.key";
```

Comment:
Will there be a way for generated clients to generate these?  @m-nash  FYI

Reply:
Good thought. As a default, perhaps it needs to be the package name with acceptable translations like "Azure" to "az" as a prefix or PascalCase to pascal_case, but then you end up with "az.data.app_configuration" as a default. Perhaps an optional tspconfig parameter to change it or some other means?

Reply:
The attributes in `az.` namespace are custom things and are always populated manually. 
Maybe at some point we'll have a typespec feature to declare attributes for specific operations, but if we do this, we'd probably just declare each name independently.

For now, let's say that generated code should not set any attributes.